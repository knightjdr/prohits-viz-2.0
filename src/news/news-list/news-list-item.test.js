import React from 'react';
import { shallow } from 'enzyme';

import NewsListItem from './news-list-item';

const item = {
  _id: 'test',
  date: 'test',
  details: 'test',
  headline: 'test',
};

describe('NewsListItem', () => {
  it('should render and match snapshot', () => {
    const wrapper = shallow(
      <NewsListItem
        item={item}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
