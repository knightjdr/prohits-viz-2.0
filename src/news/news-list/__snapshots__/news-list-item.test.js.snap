// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsListItem should render and match snapshot 1`] = `
<Item
  actions={
    Array [
      <Unknown
        to="/news/test"
    >
        more
    </Unknown>,
    ]
  }
>
  <Meta
    description={
      <div>
        <div>
          test
        </div>
        <div
          className="news__list-details"
        >
          test
        </div>
      </div>
    }
    title="test"
  />
</Item>
`;
